posthog/migrations/0156_insight_short_id.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.1.8 on 2021-05-26 23:18
+
+from django.db import migrations, models
+
+import posthog.models.dashboard_item
+
+
+def create_short_ids(apps, schema_editor):
+    DashboardItem = apps.get_model("posthog", "DashboardItem")
+    for obj in DashboardItem.objects.all():
+        obj.short_id = posthog.models.dashboard_item.generate_short_id()
```

Comment:
This will likely cause users to be unable to update in the future if we ever rename the method in the model

Rather than relying on app code in a migration it's smarter to inline the code, even if it means duplication.